jest.config.js

```diff
@@ -5,6 +5,12 @@ module.exports = {
   // Automatically clear mock calls and instances between every test
   clearMocks: true,
 
+  // Automatically reset mock state before every test
```

Comment:
I don't think we should reset and restore all mocks in the global config because most of other tests in our suite are using mocks. It does break a lot of tests if we run `yarn test`.

Instead I think we can just call `jest.resetAllMocks()` within the first `beforeEach()` to copy the behaviour just to this file. I don't think we need the `restoreAllMocks` because the documentation says it only effects `jest.spyOn` created mocks.

Reply:
Just adding to Max. 

If I understand correctly, it seems the only place that needs resetting is `mockFromFetch`. So, we can call `mockFromFetch.mockReset();`  instead to limit the impact.

Reply:
I'll remove these. I think this was from trying to ensure that LoginService didn't get mocked in my tests. I was able to just use `jest.unmock`, but couldn't figure out why it was being mocked in the first place.

Reply:
Thanks for pointing that out!